pkg/controller/jenkins/configuration/user/seedjobs/seedjobs.go

```diff
@@ -160,7 +162,21 @@ func (s *SeedJobs) EnsureSeedJobs(jenkins *v1alpha2.Jenkins) (done bool, err err
 	}
 
 	if len(jenkins.Spec.SeedJobs) > 0 {
-		err := s.createAgent(s.jenkinsClient, s.Client, jenkins, jenkins.Namespace, AgentName)
+		requeue, err := s.ensureAgentExecutorsAmount()
+		if err != nil {
+			return false, err
+		}
+		if requeue {
+			return false, nil
+		}
```

Comment:
remove and set default value here https://github.com/jenkinsci/kubernetes-operator/blob/master/pkg/controller/jenkins/jenkins_controller.go#L327